installer/install.iss

```diff
@@ -308,11 +308,12 @@ const
     GE_Nano           = 0;
     GE_VIM            = 1;
     GE_NotepadPlusPlus = 2;
-    GE_VisualStudioCode = 3;
-    GE_VisualStudioCodeInsiders = 4;
-    GE_SublimeText = 5;
-    GE_Atom = 6;
-    GE_CustomEditor = 7;
+    GE_VSCodium       = 3;
+    GE_VisualStudioCode = 4;
+    GE_VisualStudioCodeInsiders = 5;
+    GE_SublimeText    = 6;
+    GE_Atom           = 7;
+    GE_CustomEditor   = 8;
```

Comment:
I have the impression that you want to use Git for Windows to further a certain agenda to push VSCodium. Git for Windows is not the right venue for that, though. It would be abusing a highly visible software project for your own ends rather for the ends of said project.

To avoid any problematic outcome, I decided to rework your patch and merge it _without_ the changes unrelated to VSCodium, and I also kept the original order of the editors: the popularity of the supported editors matters much more to Git for Windows' users than other considerations.

Reply:
> I have the impression that you want to use Git for Windows to further a certain agenda to push VSCodium.

Be sure that this is not the case, I thought I'd follow one of Git's goals (see below).

> the popularity of the supported editors matters much more to Git for Windows' users than other considerations

This was the point where my misunderstanding was. I've thought that it the user's freedom is considered more important and that by the given order it also looks like that you "suggest" the user to _install_ VSCode.

> To avoid any problematic outcome, I decided to rework your patch and merge it without the changes unrelated to VSCodium, and I also kept the original order of the editors [...]

I agree that this seems the most reasonable thing to do for now. Thank you for the additional effort you've done.